models/Students.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace app\models;
+
+use Yii;
+
+/**
+ * This is the model class for table "students".
+ *
+ * @property integer $id
+ * @property string $student_name
+ * @property integer $department_id
+ *
+ * @property Department $department
+ * @property StudentsHomework[] $studentsHomeworks
+ * @property Homework[] $homeworks
+ */
+class Students extends \yii\db\ActiveRecord
```

Comment:
Student, not Students. The table can be named "students" or "student" but the model should be "Student"